examples/cord_lc/cord_lc_cli.c

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (C) 2018 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ * @brief       Shell command for the cord_lc module
+ *
+ * @author      Aiman Ismail <muhammadaimanbin.ismail@haw-hamburg.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "net/cord/config.h"
+#include "net/cord/lc.h"
+#include "net/gcoap.h"
+#include "net/sock/util.h"
+
+static cord_lc_rd_t rd;
+static sock_udp_ep_t remote;
+static char rdbuf[2 * NANOCOAP_URI_MAX] = {0};
+static unsigned rd_initialized = 0;
+
+static int _make_sock_ep(sock_udp_ep_t *ep, const char *addr) {
+    ep->port = 0;
+    if (sock_udp_str2ep(ep, addr) < 0) {
+        return -1;
+    }
+    ep->family = AF_INET6;
```

Comment:
is already set by `sock_udp_str2ep` according to given `addr` string, and should not be manually be overriden, bc in theory this could also work with IPv4.

Reply:
fixed